### DISCUSSION TITLE
Is there any optimization / finetuning file for paddle OCR?

### DISCUSSION THREAD

**Discussion by Haseeeb21** [PaddlePaddle/PaddleOCR#13967] [General]
Is there any optimization / fine tuning file shared in the repository?

On how can we optimize the model and how can we fine-tune the model?

Kindly if there is any code file available do share the link and the documentation.

**Comment by GreatV** [comment]
please refer to https://paddlepaddle.github.io/PaddleOCR/latest/en/ppocr/model_train/finetune.html
